README.md

```diff
@@ -1,74 +1,52 @@
 # [Prettier](https://prettier.io/) Plugin for [Astro](https://astro.build/)
 
-Official Prettier plugin adding support for formatting `.astro` files
+Official Prettier plugin adding support for formatting `.astro` files.
 
 ## Installation
 
 ```shell
 npm i --save-dev prettier-plugin-astro prettier
 ```
 
-To customize formatting behavior, see the [Configuration](#configuration) section below
+### Recommended configuration
 
-## Using with the Prettier CLI
+For optimal compatibility with the different package managers and Prettier plugins, we recommend manually specifying your plugins and the parser to use for Astro files in your Prettier config, like such:
 
-When using the CLI, Prettier will automatically pick up the plugin
-
-```shell
-prettier -w .
+```js
+module.exports = {
+  plugins: [require.resolve('prettier-plugin-astro')],
+  overrides: [
+    {
+      files: '*.astro',
+      options: {
+        parser: 'astro',
+      },
+    },
+  ],
+};
 ```
 
-### pnpm support
+The `require.resolve` call can alternatively be changed to a direct path, like such: `plugins: ["./node_modules/prettier-plugin-astro"]` for usage inside a non-JS config file.
```

Comment:
```suggestion
Alternatively, for use inside a non-JS config file, the `require.resolve` call can be changed to a direct path, such as `plugins: ["./node_modules/prettier-plugin-astro"]`.
```
Just front-loading the why they might need to know this part, highlighting that it's an alternative (not something extra they need to do/know) and when to consider this alternative.